src/test/java/com/vertexvis/model/AnyOfMetadataLongTypeMetadataFloatTypeMetadataDateTypeMetadataStringTypeMetadataNullTypeTest.java

```diff
@@ -0,0 +1,150 @@
+package com.vertexvis.model;
+
+import com.vertexvis.JSON;
+import org.junit.jupiter.api.Test;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.Objects;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class AnyOfMetadataLongTypeMetadataFloatTypeMetadataDateTypeMetadataStringTypeMetadataNullTypeTest {
+
```

Comment:
Tests support for serialization and deserialization of metadata property values.